code/sorting/quick_sort/quick_sort.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+functionfunction quick_sort( $array ) {
```

Comment:
Is this a syntax error? Why is `function` written twice?

Reply:
I don't know how that happend but it is fixed